third_party/terraform/utils/node_config.go.erb

```diff
@@ -148,6 +148,24 @@ var schemaNodeConfig = &schema.Schema{
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 
+			"shielded_instance_config": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
Do either of these have defaults from the API? Having both optional allows for
```
{
...
shielded_instance_config {}
...
}

which is fine but weird.

Reply:
I didn't see anything documented, but I see the default node pool's `enable_integrity_monitoring` returns true by default.  Should I set that to true and `secure_boot` to false do you think?

Reply:
I think you should set `enable_integrity_monitoring` to true by default and document that in the description.